src/plugins/intel_cpu/src/transformations/transformation_pipeline.cpp

```diff
@@ -358,7 +358,9 @@ void Transformations::PreLpt(const std::vector<ov::element::Type>& defaultPrecis
     decompression_handling_manager.set_per_pass_validation(false);
     CPU_REGISTER_PASS_COMMON(decompression_handling_manager, ov::pass::InitNodeInfo);
     const bool useLpt = !defaultPrecisions.empty();
-    CPU_REGISTER_PASS_COMMON(decompression_handling_manager, ov::pass::ConvertGatherToGatherCompressed);
+    auto gather_decompression = decompression_handling_manager.register_pass<ov::pass::GraphRewrite>();
+    ADD_MATCHER(gather_decompression, ov::pass::CompressedGatherTransformation);
+    gather_decompression->set_name("ov::pass::GatherDecompression");
```

Comment:
We don't need to create a separate `GraphRewrite` here, since `CompressedGatherTransformation` is already `GraphRewrite`. Can we just register it using `CPU_REGISTER_PASS_COMMON` macro?

Reply:
done, thanks Vladislav!